ack-eliciting not ACK-eliciting (#3069)
quicwg/base-drafts: draft-ietf-quic-transport.md

: A complete processable unit of QUIC that can be encapsulated in a UDP
  datagram.  Multiple QUIC packets can be encapsulated in a single UDP datagram.

Ack-eliciting Packet:

> : A QUIC packet that contain frames other than ACK and PADDING elicits an ACK

Reviewer: ```suggestion
: A QUIC packet that contains frames other than ACK and PADDING. These cause a recipient
   to send an acknowledgment within the maximum ack delay.
```

>   recipient to send an acknowledgment within the maximum ack delay.

Reviewer 2: ```suggestion
  recipient to send an acknowledgment within its maximum ACK delay ({{transport-parameter-definitions}}).
```

: A complete processable unit of QUIC that can be encapsulated in a UDP
  datagram.  Multiple QUIC packets can be encapsulated in a single UDP datagram.

Ack-eliciting Packet:

: A QUIC packet that contains frames other than ACK and PADDING. These cause a

>   recipient to send an acknowledgment within its maximum ACK delay

Reviewer 2: ```suggestion
  recipient to send an acknowledgment (see {{sending-acknowledgements}}).
```

>   ({{transport-parameter-definitions}}).

Reviewer 2: ```suggestion
```

Packets containing only ACK frames are not congestion controlled, so there are
limits on how frequently they can be sent.  An endpoint MUST NOT send more than
one ACK-frame-only packet in response to receiving an ack-eliciting packet (one
containing frames other than ACK and/or PADDING).  An endpoint MUST NOT send a

> packet containing only an ACK frame in response to a non-acl-eliciting packet

Reviewer 3: s/acl/ack
